symphony/lib/toolkit/class.section.php

```diff
@@ -123,10 +139,19 @@ public function getSortingOrder()
      */
     public function setSortingField($sort, $write = true)
     {
-        Symphony::Configuration()->set('section_' . $this->get('handle') . '_sortby', $sort, 'sorting');
+        Symphony::ExtensionManager()->notifyMembers('setSortingField', '/publish/', array(
+            'section-handle' => $this->get('handle'),
+            'sort' => $sort,
+            'updated' => $updated,
```

Comment:
1. I think 'sort' should be 'field' for consistency. I would also simply pass the $order by ref and drop the 'updated' key.